config/install/user.role.az_user_admin.yml

```
@@ -0,0 +1,11 @@
+langcode: en
+status: true
+dependencies: {  }
+id: az_user_admin
+label: 'User administrator'
+weight: 4
+is_admin: false
+permissions:
+  - 'administer users'
+  - 'assign az_content_admin role'
+  - 'assign az_content_editor role'
```

Do we need to add this role as well?
```suggestion
  - 'assign az_content_editor role'
  - 'assign az_user_admin role'
```